keyboards/ploopyco/trackball_thumb/keymaps/drag_scroll/keymap.c

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 Colin Lam (Ploopy Corporation)
+ * Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+ * Copyright 2019 Sunjun Kim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT( /* Base */
+        KC_BTN4, KC_BTN1, KC_BTN3, KC_BTN2, KC_BTN5,
+        LT(1, KC_BTN4)
```

Comment:
I believe `LT` doesn't work with mouse buttons, as they're not basic keycodes.

Reply:
Technically they are not "basic" keycodes, however they currently live in the basic keycode range, so this will work for the time being.

Reply:
Would it make sense to not include this map, then? Or put it into a future PR that depends on #16076? It seems weird to have something as an example that's known not-working. Maybe a comment, at least?

Reply:
fixed